src/nxdoc/nuxeo-server/nuxeo-server-release-notes.md

```diff
@@ -849,14 +1753,37 @@ As a consequence, the user might not have the video info in the UI immediately a
 
 <i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-24316](https://jira.nuxeo.com/browse/NXP-24316).
 
+#### Operation to Recompute Image Renditions {{since '10.3'}}
+
+You may want to recompute your stored image renditions when you change the resolutions you want to use. A new operation is available for this: `Picture.RecomputeViews`.
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-25791](https://jira.nuxeo.com/browse/NXP-25791).
+
+#### Support for Digital Moving-Picture Exchange (DPX) Format  {{since '10.10'}}
+
+DPX files (video production format) are now supported, including dragn'drop (recognized as a video) and thumbnail generation.
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-26448](https://jira.nuxeo.com/browse/NXP-26448).
+
```

Comment:
Might also want to mention GXF and MXF video formats: https://jira.nuxeo.com/browse/NXP-25896